pkg/security/security.go

```diff
@@ -38,6 +38,8 @@ const (
 	// DefaultRootCertFilePath is the well-known path for an existing root certificate file
 	DefaultRootCertFilePath = "./etc/certs/root-cert.pem"
```

Comment:
Is this still valid ? Isn't the current default the mounted volume ( to be replaced by an injected SHA in 1.10 or so  )?